src/OrchardCore.Modules/OrchardCore.Contents/Views/Stereotype.PublishButton.cshtml

```diff
@@ -0,0 +1,5 @@
+@{
+    Model.Metadata.Type = "Content_PublishButton";
```

Comment:
You don’t need create this file, it’s not related to stereotype

Reply:
done